tests/ecosystem/deployment/test_deployment.py

```diff
@@ -162,3 +163,13 @@ def ocs_install_verification():
         item['metadata']['name'] for item in storage_classes['items']
     }
     assert required_storage_classes.issubset(storage_class_names)
+
+    # Verify OSD's are distributed
+    log.info("Verifying OSD's are distributed evenly across worker nodes")
+    ocp_pod_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
Instead of defaults.ROOK_CLUSTER_NAMESPACE use cluster name from config.ENV_DATA section

Reply:
Done